### PR TITLE
Setup ci for  redhat-subscription-injection-webhook

### PR DESCRIPTION
**Pull request by coreydaley**
Setup CI for openshift/subscription-injection-webhook

### CODE REVIEW

**Comment by bparees** [comment]
only @smarterclayton can approve the mirror update.
only @openshift/openshift-team-developer-productivity-test-platform can approve the plugin part

**Comment by coreydaley-redhat** [comment]
@openshift/openshift-team-developer-productivity-test-platform ptal

**Review by petr-muller — APPROVED** [review]

File: ci-operator/config/openshift/subscription-injection-webhook/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+reviewers:
+- otaviof
+- coreydaley
```
Comment (petr-muller) [inline ci-operator/config/openshift/subscription-injection-webhook/OWNERS:3]: Should this be coreydaley-redhat ?

Comment (coreydaley-redhat) [reply ci-operator/config/openshift/subscription-injection-webhook/OWNERS]: updated in current push.

**Comment by coreydaley-redhat** [comment]
@petr-muller I have resolved the OWNERS file issue.

**Review by petr-muller — APPROVED** [review]

**Comment by smarterclayton** [comment]
What does this component do?  What kind of subscription is being injected?

**Comment by bparees** [comment]
> What does this component do? What kind of subscription is being injected?

rhel entitlements

**Comment by bparees** [comment]
@smarterclayton see:
https://github.com/openshift/enhancements/blob/master/enhancements/subscription-content/subscription-content-access.md

for an overview and 
https://github.com/openshift/enhancements/blob/master/enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md
https://github.com/openshift/enhancements/pull/389

for some lower level details

**Comment by smarterclayton** [comment]
The name is a bit too generic at first blush.  Is there any other type of subscription that could ever be injected?

**Comment by coreydaley-redhat** [comment]
@adambkaplan would probably have a better handle on the answer to that question.

**Comment by adambkaplan** [comment]
> The name is a bit too generic at first blush. Is there any other type of subscription that could ever be injected?

Not that I'm aware of today. But given that OLM uses similar terminology ("subscription"), I can't rule it out for the future. Perhaps we must rename to `redhat-subscription-injection-webhook`?

**Comment by coreydaley-redhat** [comment]
@smarterclayton everything has been updated with the name change, ptal

**Pull request merged**